code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -203,6 +203,23 @@
 
 	return ..()
 
+datum/reagent/romerol
+	name = "romerol"
+	// the REAL zombie powder
+	id = "romerol"
+	description = "Romerol is a highly experimental bioterror agent \
+		which causes dormant nodules to be etched into the grey matter of \
+		the subject. These nodules only become active upon death of the \
+		host, upon which, the secondary structures activate and take control \
+		of the host body."
+	color = "#123524" // RGB (18, 53, 36)
+	metabolization_rate = INFINITY
+
```

Comment:
Might as well add a taste message here about it tasting like brains, now that the taste PR is in

Also missing can_synth = FALSE.

Would be a bad thing if this is mass produced.

Reply:
Added in a walking dead reference